CMakeLists.txt

```diff
@@ -309,6 +309,8 @@ OCV_OPTION(WITH_OPENJPEG "Include JPEG2K support (OpenJPEG)" ON
 OCV_OPTION(WITH_JPEG "Include JPEG support" ON
   VISIBLE_IF TRUE
   VERIFY HAVE_JPEG)
+OCV_OPTION(WITH_LIBJPEG_TURBO_SIMD "Include SIMD extensions for libjpeg-turbo, if available for this platform" (NOT CV_DISABLE_OPTIMIZATION)
+  VISIBLE_IF BUILD_JPEG)
```

Comment:
Please move this flag into `3rdparty/libjpeg-turbo`

`WITH_` prefix should be replaced to `ENABLE_`
(`WITH_` is used in OpenCV for "auto-detection" of external components/dependencies/libraries)